float32 l_speed
float32 r_speed
float32 servo_pos